Update rxjs and redux-observable

gmarcos87 opened the pull request:
These changes adapt the source code to the decision to remove _ActionsObservable_ in favor of just _Observable_ in **redux-observable**.
The app was currently running fine in development mode but at the time of the production build the **rxjs** prototypes failed. This is solved by this pull-request.

gmarcos87 committed:
fix(bug): 10.5.0.1 as home address

gmarcos87 committed:
build(preact): fix build command

gmarcos87 committed:
build(rxjs): update to new api

gmarcos87 committed:
feat(rxjs): update to new api in epics and api files

spiccinini commented on package.json:
```diff
@@ -9,7 +9,7 @@
     "start:production": "npm run -s serve",
     "start:development": "npm run -s dev",
-    "build": "preact build --no-prerender && rm build/*.map",
+    "build": "preact build --no-prerender --production --no-esm && rm build/*.map",
```
This hardcodes "production only" builds? The other day I had to remove the --prduction and the rm *.map. Maybe we can have a "build_development" command?

gmarcos87 replied:
Hi @spiccinini, How nice to see you here too :) 
I added _npm run build:production_

gmarcos87 committed:
ci(node): change node version

gmarcos87 committed:
ci(build): add build:production

tihs remove the --prduction and the rm *.map from npm run build
and adds npm run build:production

germanferrero commented on plugins/lime-plugin-metrics/src/metricsApi.js:
```diff
@@ -1,11 +1,14 @@
-export const getGateway = (api, sid) => api.call(sid, 'lime-metrics', 'get_gateway', {})
-	.map(result => result);
+export const getGateway = (api, sid) => api.call(sid, 'lime-metrics', 'get_gateway', {}).pipe(
+	map(result => result)
```
Is this map call necessary here? I'm not familiar with the API.

gmarcos87 replied:
Nice catch!

gmarcos87 replied:
fixed on 49e4457

germanferrero commented on plugins/lime-plugin-notes/src/notesPage.js:
```diff
@@ -36,9 +36,9 @@ export const Page = ({ setNotes, getNotes, notes, hostname, loading }) => {
+		<div class="container" style={{ paddingTop: '100px' }}>
 			<h4><span>{I18n.t('Notes of')}</span> {hostname}</h4>
-			<textarea onChange={handleChange} className={style.notes} value={value} />
+			<textarea onChange={handleChange} class={'notes'} value={value} />
```
We should keep className instead of class since class is a reserved word in js, shouldn't we?

gmarcos87 replied:
There seems to be a problem with importing styles and if I use className it does not apply the style to the textarea.

germanferrero commented on plugins/lime-plugin-metrics/src/components/box/index.js:
```diff
@@ -73,10 +73,12 @@ const Box = ({ station, settings, loading, click, gateway }) => {
-				</div>)
+				: station.bandwidth
+					? (<div>
+						{station.bandwidth || 0} Mbps / <span>{I18n.t('Packet loss')}</span> {station.loss}%<br />
```
`{ station.bandwidth || 0 }` This will always resolve to station.bandwith because of previous `?` statement

gmarcos87 replied:
fixed on bf8eb34

germanferrero reviewed (approved):
I have added some minor comments. The PR looks solid.

gmarcos87 committed:
perf(rxjs): remove useless function

gmarcos87 committed:
perf(ui): station.bandwith will never be undefined
